### DISCUSSION TITLE
Chat about your vision for this app

### DISCUSSION THREAD

**Discussion by ryankauk** [spensbot/captivate#16] [General]
Hey, I recently started getting into dmx and had the same "clunky" feeling you have pointed out and being able to play lights. I have some aspects that I'm looking for and went down a rabbit hole of looking if there was a need to build something then came across your repo. What you happen to be into connecting over discord or something to chat about your vision with the app, do some knowledge sharing and talk about the capabilities? Would love to contribute.

**Comment by Mattallmighty** [comment]
Hey Ryan!
Welcome and glad to see your are interested in Captivate! I am one of the devs working on it and looking forward to see what's next.
Discord channel: https://discord.gg/JjG9fywdNf 
See you there! Matt
